### DISCUSSION TITLE
Problema al calificar la Task 3 Lab 3.01

### DISCUSSION THREAD

**Discussion by JohanSH7** [rramosp/ai4eng.v1#285] [03 - GENERACION DE MODELOS]
Al intentar correr el codigo para ver la calificacion me sale un error que creo que no tiene que ver con mi respuesta en el espacio de la tarea.
![image](https://github.com/rramosp/ai4eng.v1/assets/87247883/7dfed60a-08d9-48e7-83d8-c8172996e563)

https://colab.research.google.com/drive/16TD7r46EYrZuhclANbVD_8YM1ZvZrCtS?usp=sharing

**Comment by Felipe-RA** [comment]
Saludos @JohanSH7 

Efectivamente el error esta en tu código.

Revisa por favor el formato de lo que se espera que envíes en esa, lo que está pasando es que no encuentra el codigo fuente a evaluar (porque no estas enviandole lo que espera el calificador automático)
